components/ReserveForUserModal.tsx

```diff
@@ -0,0 +1,100 @@
+'use client';
+
+import { getProfile } from '@/supabase/models/getProfile';
```

Comment:
unfortunately this import here is causing some trouble because it's turned the integration tests on our component forcibly into end to end tests - they fail unless we spin up the entire environment.

We need to [mock](https://storybook.js.org/docs/writing-stories/mocking-data-and-modules/mocking-modules) this or move the import out and find a way to pass it within a handler prop (which I think we already saw was not a good option) - the pr won't be able to merge if this test is failing 🙃

Reply:
Yes I looked into this a little while ago but couldn't get the mock to work - may need some help here 😬